homework-submission/react-hyf/react-hyf-client/src/React_HW3/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
For this route I have the following warnings:

<img width="593" alt="Screen Shot 2019-07-20 at 21 22 23" src="https://user-images.githubusercontent.com/25272822/61583163-93554980-ab34-11e9-9ad4-79da07221674.png">

Reply:
Let me check